ui/vic-ui-h5c/vic-service/src/main/java/com/vmware/vic/PropFetcher.java

```diff
@@ -242,6 +307,36 @@ public ResultItem getVicVms(boolean isVch) {
     }
 
 	/**
+	 * Process List<DynamicProperty> on a ResourcePool and
+	 * return a map containing VM ManagedObjectReference's "value" to
+	 * its parent object's name information 
+	 * @param dpList : List<DynamicProperty>
+	 * @param map : HashMap<String, String> to contain mapping info 
+	 * @return populated map with the provided dpList
+	 */
+	private Map<String, String> getVcMorValueRpNameMap(
+	        List<DynamicProperty> dpList,
+	        Map<String, String> map) {
+	    String resourcePoolName = null;
```

Comment:
I'd probably prefer to keep the content having 4-space indents which came right from line 317